cleanup-oracle.sh

```diff
@@ -206,6 +214,10 @@ fi
     echo "Incorrect parameter provided for ora-data-mounts-json: $ORA_DATA_MOUNTS_JSON"
     exit 1
 }
+[[ ! "$REMOVE_AUTOGCS_BUCKET" =~ $REMOVE_AUTOGCS_BUCKET_PARAM ]] && {
```

Comment:
We need to handle the case where $REMOVE_AUTOGCS_BUCKET is blank too.  (see sample output: https://oss.gprow.dev/view/gs/bmaas-testing-prow-bucket/pr-logs/pull/google_oracle-toolkit/215/oracle-toolkit-install/1895185501892120576)